ethereum/spec/src/main/java/tech/pegasys/teku/spec/constants/Domain.java

```diff
@@ -30,4 +30,7 @@ public class Domain {
   public static final Bytes4 SYNC_COMMITTEE = Bytes4.fromHexString("0x07000000");
   public static final Bytes4 SYNC_COMMITTEE_SELECTION_PROOF = Bytes4.fromHexString("0x08000000");
   public static final Bytes4 CONTRIBUTION_AND_PROOF = Bytes4.fromHexString("0x09000000");
+
+  // EIP-4844
+  public static final Bytes4 BLOBS_SIDECAR = Bytes4.fromHexString("0x0a000000");
```

Comment:
I just realized this has been removed today from spec :)
makes sense since blobs are not signed anymore...
So we should remove it from here and in the other places.

Reply:
Ahaha funny, I've done this just before removal.
Ok.